IntelliJ Gradle task is incorrect

Docs call for running `gradle intellij` to generate *.iml and .idea/ files for IntelliJ import - with result:

```
λ derpy etl → λ git master* → gradle intellij

FAILURE: Build failed with an exception.

* What went wrong:
Task 'intellij' not found in root project 'etl'.

* Try:
Run gradle tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with                                               --info                                                                           or                                                                             --debug                                                                          option to get more log output.

BUILD FAILED
```

Running `gradle tasks` shows the correct task for this operation is `gradle idea`. 

PR forthcoming shortly.

--- a/app/pages/languages/getting-started-with-java.md
+++ b/app/pages/languages/getting-started-with-java.md
@@ -63,9 +63,9 @@ For Eclipse,
 
 generates the `.project` file and `.settings/` directory. In Eclipse, import this project into your workspace: File > Import > Existing Projects Into Workspace.
 
-For IntelliJ,
+For IntelliJ IDEA,
 
-    gradle intellij
+    gradle idea
 
 generates the `<project-name>.iml` file and `.idea/` directory. In IntelliJ, open this project: File > Open.
 

